install.sh

```diff
@@ -39,6 +39,29 @@ then
   abort 'Bash must not run in POSIX mode. Please unset POSIXLY_CORRECT and try again.'
 fi
 
+usage() {
+  cat <<EOS
+Homebrew Installer
+Usage: [NONINTERACTIVE=1] [CI=1] $0 [options]
```

Comment:
```suggestion
Usage: [NONINTERACTIVE=1] [CI=1] install.sh [options]
```

Reply:
I’ve used $0 because it’s what the uninstaller uses, but I’m fine with either